### DISCUSSION TITLE
Does it always convert to youtube?

### DISCUSSION THREAD

**Discussion by DrNefarius** [codetheweb/muse#889] [Q&A] [answered]
Hey,

I am trying to get the bot to play a spotify song but every time i provide it with a link to that song it converts it to a youtube video and gives me the live version of that song (which is not what i want).
Is there something I am missing here? A configuration or something?
Im running it in a docker container and provided correct Spotify environment variables.
Also when I look at the developer dashboard from spotify I see that the number of requests is still at 0.
Cheers

**Comment by codetheweb** [comment, marked as answer]
Muse does not directly stream from Spotify, it gathers track/artist/album info from Spotify and then converts it to YouTube videos to the best of its ability. 

If you want to directly stream from Spotify, you might want to check out my other music bot: [Aoede](https://github.com/codetheweb/aoede)
